tests/scaffoldings/dotd/custom/dependencies/pip.txt

```diff
@@ -1,6 +1,5 @@
 # Compiled locally, which would fail if `apt_build.txt` fails
+pip==23.3.1
```

Comment:
the comment above this line belongs to the pycrypto pip line, if you are locking the pip version please add it as line 1 (and maybe add a comment why the version is locked)

initially each of the lines in this pip.txt worked as a separate test on their own here (as far as i remember)

Reply:
Certainly. I added this line (version 23.3.1, it's the latest version of pip) because I wanted to make sure that the pip version wouldn't cause any issues. I am testing locally to confirm if I don't need it, and if that's the case, I'll remove it. Thanks for the review!

Reply:
We are using for now an old Debian version. Another thing to do is to use bookworm instead.